app/client/components/delivery/address/deliveryAddressFormContext.tsx

```diff
@@ -0,0 +1,15 @@
+import { createContext } from "react";
+import { DeliveryAddress } from "../../../../shared/productResponse";
+import { ContactIdToArrayOfProductDetail } from "./deliveryAddressForm";
+
+export const StateResetContext = createContext<() => void>(() => true);
+
+export const DeliveryAddressContext = createContext<DeliveryAddress | {}>({});
```

Comment:
1. could these contexts be combined into one?
2. since this holds 🆕 delivery address, perhaps the name could reflect that so that nobody confuses it with holding the original delivery address?